js/interface.js

```diff
@@ -1595,6 +1601,23 @@ $('body').on('click', '[data-rule-delete]', function (event) {
   updateDataSourceRules();
 });
 
+$('body').on('click', '[data-toggle-status]', function (event) {
+  event.preventDefault();
+
+  var index = parseInt($(this).closest('tr').data('rule-index'), 10);
+  var rule = currentDataSourceRules[index];
+
+  rule.enabled = !rule.enabled;
+
+  // Briefly show a UI feedback as the rule enables/disables
+  $(this).find('i')
+    .addClass('fa-spinner')
```

Comment:
Do you perhaps need the `fa-pulse` class as well so it's animated? Otherwise it could look like a spinner that's stuck. https://fontawesome.com/v4.7.0/examples/

```suggestion
    .addClass('fa-spinner fa-pulse')
```

Reply:
@tonytlwu done 👍